python/examples/asset_agnostic_rules/main.py

```diff
@@ -56,21 +57,33 @@
             sub_expressions=[
                 SubExpression("voltage.overvoltage", {"$1": 75}),
                 SubExpression("voltage.undervoltage", {"$1": 30}),
-                SubExpression(
-                    "velocity.vehicle_stuck",
-                    {
-                        "$1": "vehicle_state",
-                        "$2": "mainmotor.velocity",
-                    },
-                ),
-                SubExpression(
-                    "velocity.vehicle_not_stopped",
-                    {
-                        "$1": "vehicle_state",
-                        "$2": "10",
-                    },
-                ),
+                SubExpression("velocity.vehicle_not_stopped", {"$2": 10}),
             ],
+            channel_references_map={
```

Comment:
Is it possible to keep references for a rule colocated like the original? It makes it a lot easier to look back and forrth between the .yml and here to see the substitutions and understand the rule.

Maybe something like this:
```
                SubExpression(
                    "velocity.vehicle_not_stopped",
                    {
                        "$1": ExpressionChannelReference("vehicle_state"),
                        "$2": "10",
                    },
                ),
```

Reply:
Sure I think that makes sense!

Reply:
Addressed in [080e763](https://github.com/sift-stack/sift/pull/146/commits/080e763e87e7c6891bbd028672813277a02d59b5)!